graphene/src/layers/style/mod.rs

```diff
@@ -1,6 +1,9 @@
 use crate::color::Color;
 use serde::{Deserialize, Serialize};
 const OPACITY_PRECISION: usize = 3;
+pub const WIRE_FRAME_STROKE_WIDTH: f32 = 1.0;
+pub const EMPTY_FILL: Fill = Fill::none();
+pub const THIN_BLACK_STROKE: Stroke = Stroke::new(Color::BLACK, WIRE_FRAME_STROKE_WIDTH);
```

Comment:
Perhaps change to WIREFRAME_FILL and WIREFRAME_STROKE to be more descriptive about what it is used for (and to make sense if the wireframe colour is changed)

Reply:
I changed THIN_BLACK_STROKE to WIREFRAME_STROKE (tbh that was a kinda weird sounding name), but I kept EMPTY_FILL because it is a fairly descriptive name which may be used for other things besides wireframe.